package.json

```diff
@@ -62,5 +62,18 @@
     "rollup-watch": "^4.3.1",
     "serve": "6.5.8",
     "uglify-js": "^3.3.28"
+  },
+  "jspm": {
+    "files": [
+      "package.json",
+      "bower.json",
```

Comment:
There is no `bower.json`.

Reply:
Fair comment - I copy/pasted from the existing `files` section in `package.json`, is that also wrong?

Reply:
Oh, yeah. The entry is actually obsolete.

Reply:
Solved 4688a5fb211b338224d6f47b99684fcee138320e

Reply:
OK. I'll remove both entries in my branch.

Reply:
Maybe it's better if you merge the commit into your branch.

Reply:
Didn't see your commit / comment (I'm using my phone), will do.